tests/Foundation/FoundationInteractsWithMailableTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Mockery as m;
```

Comment:
This is a convention you'll find in all the framework's tests, I'm not sure why though :)